src/Providers/ScoutApmServiceProvider.php

```diff
@@ -99,6 +115,10 @@ public function boot(Kernel $kernel, ScoutApmAgent $agent, FilteredLogLevelDecor
     {
         $log->debug('Agent is starting');
 
+        $this->publishes([
+            __DIR__ . '/../../config/scout.php' => config_path('scout.php'),
```

Comment:
In Ruby/Rails we use `scout_apm.yml` for the config file name. I'd suggest we use `scout_apm.php` here.

Reply:
Yep; this is moved to #37 already :+1: